gson/src/test/java/com/google/gson/functional/ReflectionAccessTest.java

```diff
@@ -113,6 +114,8 @@ public void testSerializeInternalImplementationObject() {
     try {
       gson.fromJson("{}", internalClass);
       fail("Missing exception; test has to be run with `--illegal-access=deny`");
+    } catch (JsonSyntaxException e) {
+      fail("Unexpected exception; test has to be run with `--illegal-access=deny`");
```

Comment:
Not directly related to this PR, but it makes test failures for this test method when not running with `--illegal-access=deny` easier to understand.